### PR TITLE
AUTH-282: SSO Configuration for ACS

### PR DESCRIPTION
**Pull request by rpopa**
- added enforce SAML login steps

### CODE REVIEW

File: docs/config/ping-federate-example.md [hunk 1]
```diff
@@ -170,3 +170,15 @@ This section allows an existing SAML user to be automatically created in Identit
     User attribute Name: LastName
     ```
  
+### Enforce SAML Login in Identity Service
+This section allows you to **enforce SAML login**: If you do not enforce the SAML login, the user can decide to use either the Alfresco login or login using the IdP link.
```
Comment (rgauss) [inline docs/config/ping-federate-example.md]: "... the user can decide to use either the **Identity Service** login or..."

File: docs/config/ping-federate-example.md [hunk 2]
```diff
@@ -170,3 +170,15 @@ This section allows an existing SAML user to be automatically created in Identit
     User attribute Name: LastName
     ```
  
+### Enforce SAML Login in Identity Service
+This section allows you to **enforce SAML login**: If you do not enforce the SAML login, the user can decide to use either the Identity Service login or login using the IdP link.
+In order to enable it follow this steps:
```
Comment (aymanthefirst) [inline docs/config/ping-federate-example.md]: **these** instead of **this**

File: docs/config/ping-federate-example.md [hunk 3]
```diff
@@ -170,3 +170,15 @@ This section allows an existing SAML user to be automatically created in Identit
...
+
+1. Sign in to the administrator panel of the Identity Service.
+2. Select **Authentication** from side options
+3. Locate and click on **Flows** tabe
```
Comment (aymanthefirst) [inline docs/config/ping-federate-example.md]: **tab** instead of **tabe**

File: docs/config/ping-federate-example.md [hunk 4]
```diff
@@ -170,3 +170,15 @@ This section allows an existing SAML user to be automatically created in Identit
...
+4. From the drop down list select **Browser**
+5. Locate **Identity Provider Redirector** as **Auth Type** and press the *Action*/*Config* link
+6. Fill the **Alias** and **Description** fields with the identity provider you configured above (in this case *saml* for both) and save form.
```
Comment (aymanthefirst) [inline docs/config/ping-federate-example.md]: save **the** form

File: docs/config/ping-federate-example.md [hunk 5]
```diff
@@ -170,3 +170,15 @@ This section allows an existing SAML user to be automatically created in Identit
...
+
+Now the SAML login should be enforced and user is redirected straight to the IdP login page (PingFederate in this case).
```
Comment (aymanthefirst) [inline docs/config/ping-federate-example.md]: and **the** user

**Pull request merged**